config/environments/production.rb

```diff
@@ -15,7 +15,7 @@
   config.assets.compress = true
 
   # Don't fallback to assets pipeline if a precompiled asset is missed
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
This is already set to true in the qa environment and production. Why is it github saying otherwise?